tests/test_examples.py

```diff
@@ -74,23 +70,21 @@ def test_scripting_examples():
     """ """
 
     exclude = ["plotting.py", "__pycache__"]
-    for example in pkg.resource_listdir("oemof.tabular", "examples/scripting"):
+    examples_dir = os.path.join(
+        importlib.resources.files("oemof.tabular"), "examples/scripting"
+    )
+    for example in os.listdir(examples_dir):
         if not example.endswith(".ipynb") and example not in exclude:
             print("Running scripting example {} ...".format(example))
-            exec(
-                open(
-                    pkg.resource_filename(
-                        "oemof.tabular",
-                        "examples/scripting/{}".format(example),
-                    )
-                ).read()
-            )
+            exec(open(os.path.join(examples_dir, example)).read())
 
 
 def test_examples_datapackages_scripts_infer():
     """ """
     script = "infer.py"
 
+    import pkg_resources as pkg
```

Comment:
I do not remember, I think because the test failed originally (before opening this PR and until you merged dev to here) and I did not try hard to figure out why so I did not touch it. Thanks for fixing it!